hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import styles from "./Navbar.module.scss";
+import { Link } from "react-router-dom";
+import { connect, useSelector } from "react-redux";
+// Images and logos
+import DashboardIcon from "@material-ui/icons/Dashboard";
+import LocalMallIcon from "@material-ui/icons/LocalMall";
+import ListAlt from "@material-ui/icons/ListAlt";
+import AccountBoxIcon from "@material-ui/icons/AccountBox";
+import InsertChartOutlined from "@material-ui/icons/InsertChartOutlined";
+import { ReactComponent as Inventory } from "assets/images/icons/Hardware.svg";
+// Components
+import Button from "@material-ui/core/Button";
+
+import { logout } from "slices/users/userSlice";
+import { RootState } from "slices/store";
+import { cartSelectors } from "../../../slices/hardware/cartSlice";
+
+interface NavBarProps {
+    logout: any;
+    cartQuantity: number;
+    pathname: string;
+}
+
+export const UnconnectedNavbar = ({ logout, cartQuantity, pathname }: NavBarProps) => {
+    const cartItems = useSelector(cartSelectors.selectAll);
+    cartQuantity = cartItems.reduce((accum, item) => accum + item.quantity, 0);
```

Comment:
I don't think you need cartQuantity in the props anymore, you can remove it (remove it from the unit tests too) and declare this is a constant instead

Reply:
since we have cart quantity in 2 places (navbar and cart page) maybe it should have it's own selector, what do you think?

Reply:
sounds like a good idea